queue_job/tests/test_model_job_channel.py

```diff
@@ -46,7 +46,7 @@ def test_channel_complete_name_uniq(self):
         # not uses here due to its 'flush()' method inside it and exception raises
         # before the line 'self.env["base"].flush()'. So, we are expecting an IntegrityError.
```

Comment:
Maybe this comment has to reference new definition of flush, with the flush_all

```suggestion
        # before the line 'self.env.flush_all()'. So, we are expecting an IntegrityError.
```